src/tests/ftest/util/host_utils.py

```diff
@@ -0,0 +1,589 @@
+#!/usr/bin/python
+"""
+  (C) Copyright 2019 Intel Corporation.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+  GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+  The Government's rights to use, modify, reproduce, release, perform, display,
+  or disclose this software are subject to the terms of the Apache License as
+  provided in Contract No. B609815.
+  Any reproduction of computer software, computer software documentation, or
+  portions thereof marked with this legend must also reproduce the markings.
+"""
+from logging import getLogger
+from getpass import getuser
+import os
+from re import findall
+from socket import gethostname
+
+from avocado import fail_on
+from avocado.utils import process
+from ClusterShell.NodeSet import NodeSet, NodeSetParseError
+
+from agent_utils import AgentManager, AgentCommand, AgentYamlParameters
```

Comment:
(pylint-no-name-in-module) No name 'AgentManager' in module 'agent_utils'
(pylint-no-name-in-module) No name 'AgentCommand' in module 'agent_utils'
(pylint-no-name-in-module) No name 'AgentYamlParameters' in module 'agent_utils'